peps/pep-0729.rst

```diff
@@ -295,7 +295,8 @@ The specification has a few audiences:
 * For people who want to propose changes to the type system, it provides a
   foundation for any new proposals.
 
-Notably, the specification is not aimed at end users of typing. End users
+Notably, the specification is not aimed at end users of typing who do not
```

Comment:
I'd drop "end", otherwise it's dangerously close to the different meaning you'd get by adding a comma before "who".

As someone (not a typical user :-) who tends to care both about getting VS Code to show "no problems" *and* getting `mypy --strict` to pass I'm still skeptical about whether typical users care about multiple type checkers. Library maintainers probably have to, but they're hopefully not considered to be end users.

Reply:
Reworded this a bit more.